examples/reference/widgets/FileInput.ipynb

```diff
@@ -69,7 +80,32 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The widget also has a ``save`` method that allows saving the uploaded data to a file or [BytesIO](https://docs.python.org/3/library/io.html#binary-i-o) object."
+    "Similarly you can see the file name and mime type:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pmui.FlexBox(file_input.param.value, file_input.param.filename, file_input.param.mime_type)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Try uploading a file."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Saving the Value\n",
```

Comment:
After uploading a `.gif` file I try to save it by running the cell. But it fails

![image](https://github.com/user-attachments/assets/af931f53-3143-4b05-bcc1-201d159d3151)

```bash
TypeError: a bytes-like object is required, not 'str'
```

Reply:
The problem is that the `value` is a `str`:

![image](https://github.com/user-attachments/assets/5d9317d7-3939-47ec-bb2c-3be668f78c3b)